File: Java/Medium/WrongAnswer/rearrange-array-to-maximize-prefix-score_3.java
class Solution {
    public int maxScore(int[] nums) {
        
        int[] prefix = new int[nums.length];
        prefix[0] = nums[0];
        for (int i = 1; i < nums.length; i++) {
            prefix[i] = prefix[i - 1] + nums[i];
        }
        int max = 0;
        for (int i = 0; i < nums.length; i++) {
            if (prefix[i] > 0) {
                max++;
            }
        }
        return max;
    }
}

File: Python3/Medium/WrongAnswer/rearrange-array-to-maximize-prefix-score_3.py
class Solution:
    def maxScore(self, nums: List[int]) -> int:
        
        nums.sort()
        n = len(nums)
        prefix = [0]
        for i in range(n):
            prefix.append(prefix[-1] + nums[i])
        res = 0
        for i in range(n):
            if nums[i] > 0:
                res = max(res, i + 1 + prefix[i])
        return res